ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -93,14 +95,28 @@ Item {
             readOnly: true
             selectByMouse: true
             color: !isCurrentUser ? Theme.black : Theme.white
+            visible: contentType == 1
+        }
+
+        Image {
+            id: stickerId
+            horizontalAlignment: !isCurrentUser ? Text.AlignLeft : Text.AlignRight
+            anchors.left: parent.left
+            anchors.leftMargin: parent.chatHorizontalPadding
+            anchors.top: parent.top
+            anchors.topMargin: chatBox.chatVerticalPadding
+            width: 140
+            height: 140
+            source: contentType == 2 ? ("https://ipfs.infura.io/ipfs/" + sticker) : ""
```

Comment:
Could we have this URL come from a function call even if it returns the same value? In the future it would be interesting to allow the user to specify their own IPFS gateway instead of only being able to use infura,